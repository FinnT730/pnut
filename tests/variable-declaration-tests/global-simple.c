int x, y, z;

void testGlobalMultipleVariableDeclaration() {
  x = 4;
  y = 5;
  z = 6;
  putchar(x+48);
  putchar(10);
  putchar(y+48);
  putchar(10);
  putchar(z+48);
  putchar(10);
}

int main() {
  testGlobalMultipleVariableDeclaration();
  return 0;
}