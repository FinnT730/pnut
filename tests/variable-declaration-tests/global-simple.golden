Global Variables:
4
5
6
Global Variables Initialized:
4
5
6
